csrc/ir/utils.h

```diff
@@ -66,13 +66,20 @@ struct MatmulInputs {
   Val* bias = nullptr;
   Val* alpha = nullptr;
   Val* beta = nullptr;
+  // Input layout based on ordering of axes M,N,K, valid values.
+  // Determined based on broadcast axes.
```

Comment:
nitpick: no longer inferred from broadcast axes.

Reply:
It is determined based on which axes was broadcasted for the first Mma operand.